microsoft/vscode
# Comments view: add badge with the number of unresolved comments

Fixes #164623

Tested:
  - install the [vscode-pull-request-github](https://github.com/microsoft/vscode-pull-request-github) extension
  - open a pull request that has unresolved comments
  - open the Comments view
  - notice the badge next to the "COMMENTS" title (1 in the screenshot below)

![image](https://user-images.githubusercontent.com/485572/198114119-6ab47962-4df2-4f0d-9f0c-87fb67e59d6c.png)
